setup.py

```diff
@@ -10,8 +10,6 @@
 import io
 import os
 
-from pip._internal.network.session import PipSession
```

Comment:
I never used this one!

Reply:
No, you shouldn't - its bad habit old school things. Took me a while to figure out what was wrong. Tests just complained about missing `pip`..